src/passes/Print.cpp

```diff
@@ -242,6 +243,22 @@ struct PrintSExpression : public UnifiedExpressionVisitor<PrintSExpression> {
     return printPrefixedTypes("param", type);
   }
 
+  std::ostream& printBlockType(Signature sig) {
```

Comment:
Maybe `printControlFlowType`?

Reply:
In the spec this is called `blocktype`, so I think it is useful to match that terminology.